packages/flutter/lib/src/rendering/binding.dart

```diff
@@ -102,7 +102,7 @@ abstract class RendererBinding extends BindingBase with ServicesBinding, Schedul
 
     registerSignalServiceExtension(
       name: 'debugDumpSemanticsTreeInTraversalOrder',
```

Comment:
change traversal here as well? Also, can you change the help text that shows up when you press 'h' during a hot run to reflect the change?

Reply:
OK, I think I got that.

Reply:
I believe, the changed files were maybe not added to the PR? This is the place I was talking about: https://github.com/flutter/flutter/blob/1859e82a3dea54087dca4aca9b8e49a841b6a274/packages/flutter_tools/lib/src/resident_runner.dart#L825 

Pressing "S" no longer gives you traversal order, right?

Reply:
You're correct.  I forgot to upload to the PR.